pkg/auth/dependency/authenticator/bearertoken/deps.go

```diff
@@ -2,20 +2,21 @@ package bearertoken
 
 import (
 	"github.com/google/wire"
+
 	"github.com/skygeario/skygear-server/pkg/core/config"
 	"github.com/skygeario/skygear-server/pkg/core/db"
-	"github.com/skygeario/skygear-server/pkg/core/time"
+	"github.com/skygeario/skygear-server/pkg/time"
```

Comment:
The name `time` conflicting the stdlib `time` was always annoying to me...
Maybe rename it to `datetime`, `chrono` or something else?